src/pages/application/list/index.tsx

```diff
@@ -125,8 +136,10 @@ const ApplicationList: React.FC = () => {
 	};
 
 	const handleSetDeleteApplication = (applicationId: string) => {
-		dispatch(setApplicationId(applicationId));
-		$("#applicationDeleteModal").modal("show");
+		if (permission === "ROOT_ADMIN" || permission === "ADMIN") {
+			dispatch(setApplicationId(applicationId));
+			$("#applicationDeleteModal").modal("show");
+		}
```

Comment:
Add an `else` part with some warning message. Otherwise, user won't know what is happened.
```
if (permission === "ROOT_ADMIN" || permission === "ADMIN") {
   dispatch(setApplicationId(applicationId));
   $("#applicationDeleteModal").modal("show");
} else {
   // Send a error or warning notification
}
```